src/sentry/tasks/store.py

```diff
@@ -209,17 +209,31 @@ def _do_symbolicate_event(cache_key, start_time, event_id, symbolicate_task, dat
 
     event_id = data["event_id"]
 
+    def _continue_to_process_event():
+        process_task = process_event_from_reprocessing if from_reprocessing else process_event
```

Comment:
`from_processing` isn't defined until line 240 but this function/closure is called on line 236.  Does this not result in an exception?  Is there a way to unittest this easily?